### ISSUE TITLE
Warnings with RN 0.66.1

### ISSUE THREAD

**Issue by emoreno-94** [react-native-mapbox-gl/maps#1594]
**Describe the bug**  
When I exec `unsubscribe` method appear the next warnings
```
 WARN  EventEmitter.removeListener('MapboxOfflineRegionProgress', ...): Method has been deprecated. Please instead use `remove()` on the subscription returned by `EventEmitter.addListener`.
 WARN  EventEmitter.removeListener('MapboxOfflineRegionError', ...): Method has been deprecated. Please instead use `remove()` on the subscription returned by `EventEmitter.addListener`.
```

**To Reproduce** 
Execute `MapboxGL.offlineManager.unsubscribe(<packName>);` while a `<packName>` is downloading

**Expected behavior**  
`MapboxGL.offlineManager.unsubscribe` works without warnings

**Actual behavior**  
`MapboxGL.offlineManager.unsubscribe` works with warnings

**Versions:**  
 - Platform: Android
 - Platform OS: Android 11
 - Device: Emulator
 - Emulator/ Simulator: yes
 - Dev OS: OSX 11.6
 - react-native-mapbox-gl Version 8.4.0
 - React Native Version 0.66.1

**Comment by 1mike12** [comment]
Just wanted to add that I spent like 6 hours tearing out my hair wondering why all of a sudden my async awaits started breaking recently. Turns out it probably has something to do with this warning. Turns out that everything starts working if I comment out the import. 

My warnings however show up without even having to create the map component or do anything

```js
// import MapboxGL from '@react-native-mapbox-gl/maps';
// MapboxGL.setAccessToken(env.mapboxToken);
```
